translations/en-us.yml

```diff
@@ -847,22 +849,22 @@ institutions:
             users-by-department: 'Total Users by Department'
             public-vs-private-projects:
                 title: 'Public vs Private Projects'
-                public: 'Public Projects' 
```

Comment:
For some reason I can not make this go away.

Reply:
These seem like it's just removing the trailing space at the end of these, so it's a harmless change